### PR TITLE
Fix initial pos mol

### LINKED ISSUE #35: bug:  create_molecule(positions) only creates the fist molecule in the correct position
the optional argument `position` should take a list of list with the positions where the first particle of each molecule should be created. Currently, it only has the expected behavior for the first molecule while the rest are created at random positions

### PR DESCRIPTION
**Pull request by mariusaarsten**
Fixes #35

### CODE REVIEW

File: pyMBE.py [hunk 1]
```diff
@@ -722,101 +722,110 @@ def create_counterions(self, object_name, cation_name, anion_name, espresso_syst
...
+            used_molecules_id = self.df.molecule_id.dropna().drop_duplicates().tolist()
+            pos_index = 0 
+            for molecule_index in molecule_index_list:        
+                molecule_id = self.assign_molecule_id (name=name,pmb_type='molecule',used_molecules_id=used_molecules_id,molecule_index=molecule_index)
+                for residue in residue_list:
+                    if first_residue:
+                        if list_of_first_residue_positions == None:
+                            residue_position = None
+                        else:
+                            if len(list_of_first_residue_positions) != number_of_molecules:
```
Comment (pm-blanco) [inline pyMBE.py:763]: This sanity test should be moved to the begging of the function (there is no need to check for each loop iteration)

File: testsuite/create_molecule_position.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import numpy as np
```
Comment (pm-blanco) [inline testsuite/create_molecule_position.py:1]: note that you have added two tests that do the same (or very similar) `testsuite/create_molecule_position.py` and `testsuite/create_molecule_position_test.py` please either remove one of them or if they do something different merge them into one

**Review by pm-blanco — CHANGES_REQUESTED** [review]

**Comment by pm-blanco** [comment]
@mariusaarsten note that you need to change the name of the test in the `Makefile`, otherwise the CI crashes simply because it does not find the right file

**Comment by mariusaarsten** [comment]
@pm-blanco Done! I think the pull request should be good now

**Review by pm-blanco — APPROVED** [review]
@mariusaarsten it looks good to me now, thank you for your work!

**Pull request merged**